cooltools/api/sample.py

```diff
@@ -48,6 +49,7 @@ def sample(
     out_clr_path,
     count=None,
     frac=None,
+    cis_target=False,
     exact=False,
```

Comment:
consider adding `ignore_diags` here ? or ultimately `dist_min` and `dist_max` ?
if it does not end up oin this PR - leave an issue,  maybe ? for the future

Reply:
I'd say yes ... Also just make sure what `ignore_diags` means exactly - ignore diags to set the count target, of course - but what should happen with the ignored_diags in the subsamples matrix ? Should they be zero ?

Reply:
Yes meaning ... which of the options?
I'd say the calculation of the subsampling fraction should not take `ignore_diags` diags into account, but subsampling would be applied to all data including those diags